webpack.config.js

```diff
@@ -6,11 +6,16 @@ module.exports = {
         path: __dirname,
         filename: "built.js"
     },
-    loaders: [
-        {
-            test: /\.js$/,
-            exclude: /node_modules/,
-            loader: "babel",
-        }
-    ]
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules\/(?!acey-deucey-game-engine)/,
+                loader: "babel",
+                query: { presets: ["es2015"]}
+            }
+        ],
+        noParse: [/foundation-apps/, /lodash/]
```

Comment:
We could try adding `/angular.js$/` to this list to see if it also removes the primary Angular file from being parsed and thus speeds up your build more.